sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -503,7 +503,7 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
    */
   override def databaseExists(dbName: String): Boolean = {
     try {
-      getDatabase(dbName)
+      makeDatabase(None, dbName: String)
```

Comment:
why do we make it more verbose here?

Reply:
the readability will become worse ... I like it the way it was originally